.github/workflows/ci.yaml

```
@@ -1,4 +1,4 @@
-name: CI/CD
+¿name: CI/CD
```

actionlint (1.7.7)

unexpected key "¿name" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)